yarn/assets/service_checks.json

```diff
@@ -12,5 +12,21 @@
         ],
         "name": "Can Connect",
         "description": "Returns `CRITICAL` if the Agent cannot connect to the ResourceManager URI to collect metrics, otherwise `OK`."
+    },
+    {
+        "agent_version": "7.16.0",
```

Comment:
Is this a new service_check or were we just missing the documentation for it?

Reply:
we were missing documentation for it